docs/guide.md

```diff
@@ -0,0 +1,761 @@
+
+### Batch Jobs
+
+To check on all batch jobs:
+
+`alias sqa='squeue -o "%8u %.7a %.4D %.9P %7i %.2t %.10r %.10M %.10l %.8Q %j" --sort=P,-t,-p'`
+
+To check on your batch jobs:
+
+`alias sq='sqa -u $USER'`
+
+The output of `sq` uses several abbreviations: ST = Status, R = running, PD = pending, CG = completing.
+
+### Directories
+
+You will be working in several directories.
+
+<run_scripts_dir>: `${HOME}/E3SM/scripts`
+
+<code_source_dir>: `${HOME}/E3SM/code`
+
+<simulations_dir> differs amongst machines:
+
+- Anvil/Chrysalis (LCRC) – <simulations_dir>: `/lcrc/group/e3sm/<username>/E3SMv2`
+- Compy (PNNL) – <simulations_dir>: `/compyfs/<username>/E3SMv2`
+- Perlmutter (NERSC) – <simulations_dir>: `/global/cfs/cdirs/e3sm/<username>/E3SMv2`
+
+So, it may be useful to set the following aliases:
+
+```shell
+# Model running
+
+alias run_scripts="cd <run_scripts_dir>"
+alias simulations="cd <simulations_dir>"
+```
+
+## Configuring the Model Run – Run Script
+
+Start with an example of a run script for a low-resolution coupled simulation:  
+
+We'll use the [template script](https://github.com/E3SM-Project/E3SM/blob/master/run_e3sm.template.sh) in the E3SM repository, which uses Perlmutter.
+
+Create a new run script or copy an existing one (such as the template above). The path to it should be `<run_scripts_dir>/run.<case_name>.sh`
+
+`# Machine and project`
+
+- `readonly MACHINE=pm-cpu`: the name of the machine you’re running on.
+- `readonly PROJECT="e3sm"`: SLURM project accounting (typically `e3sm`).
+
+`# Simulation`
+
+- `readonly COMPSET="WCYCL1850"`: compset (configuration)
+- `readonly RESOLUTION="ne30pg2_r05_IcoswISC30E3r5"`: resolution. In this example, we have:
+    - `ne30pg2`: atmosphere (ne30 dynamics grid -- 30 spectral elements, pg2 physics grid)
+    - `r05`: land and river on 1/2 lat/lon grid (commonly referred to as "tri-grid")
+    - `IcoswISC30E3r5`: ocean and sea-ice on Icosahedral 30 km mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5.
+- `readonly CASE_NAME="your_casename"`: case name
+- `# readonly CASE_GROUP=""`: This will let you mark multiple cases as part of the same group for later processing (e.g., with PACE).
+
+!!! warning
+    If this is part of a simulation campaign, ask your group lead about using a `CASE_GROUP` label. Otherwise, please use a unique name to distinguish from existing `CASE_GROUP` label names, (e.g., “v2.LR“).
+
+`# Code and compilation`
+
+- `readonly CHECKOUT="latest"`: Date the code was checked out on, in the form `{year}{month}{day}`. The source code will be checked out in `<code_source__dir>/{year}{month}{day}`.
+- `readonly BRANCH="master"`: branch the code was checked out from. Valid options include “master”, a branch name, or a git hash. For provenance purposes, it is best to specify the git hash.
+- `readonly DEBUG_COMPILE=false`: option to compile with DEBUG flag (leave set to false)
+
+!!! warning
+    BEFORE RUNNING: Change `CHECKOUT` to a date string like 20240301.
```

Comment:
2 options for syntax that differs between `mkdocs` and GitHub renderings
1) optimize for `mkdocs` -- this won't render well in GitHub's markdown rendering 
2) use only the most basic syntax that works on both.

Let's go with (1)